content/en/logs/log_collection/csharp.md

```diff
@@ -35,14 +35,17 @@ To send your C# logs to Datadog, use one of the following approaches:
 - [Enable Agentless logging](#agentless-logging-with-apm).
 - [Use the Serilog sink](#agentless-logging-with-serilog-sink).
 
-This page details setup examples for the `Serilog`, `NLog`, `log4net`, and `Microsoft.Extensions.Logging` logging libraries, for each of the above approaches.
-
 ## File-tail logging with the Datadog Agent
 
 The recommended approach for C# log collection is to output your logs to a file and then [tail][20] that file with your Datadog Agent. This enables the Datadog Agent to enrich the logs with additional metadata.
 
 Datadog strongly encourages setting up your logging library to produce your logs in JSON format to avoid the need for [custom parsing rules][1].
 
+File-tail logging supports the following frameworks:
+- Serilog
+- NLog
+- log4net
```

Comment:
Is it correct that the Microsoft one got dropped in this update?

Reply:
Confirming that this wasn't dropped so much as it wasn't supported, but the sentence in the Overview section makes it sound like it was, so customers expected to see an example config for file-tail logging. It's supported for Agentless APM Logging on the same page: https://docs.datadoghq.com/logs/log_collection/csharp/?tab=serilog#agentless-logging-with-apm